app/controllers/qsets_controller.rb

```diff
@@ -13,30 +13,31 @@ def index
   # handles the request to show all questions in a qset
   def show
     @feedback_active = !!current_user
```

Comment:
Does `@feedback_active` need to be available to both views? Or just the `questions` view?